CHANGELOG.md

```diff
@@ -176,6 +176,7 @@ be used to retrieve the actual proposal `Content`. Also the `NewMsgSubmitProposa
 
 ### Bug Fixes
 
+* (x/bank) [\#6536](https://github.com/cosmos/cosmos-sdk/pull/6536) When calling bank REST endpoint to generate StdTx now returns correct JSON format.
```

Comment:
This changelog isn't really helping much IMHO. Might be better to explicitly state which route.